src/implementations/vanilla/components/site-header/site-header.js

```diff
@@ -212,6 +217,30 @@ export class SiteHeader {
     }
   }
 
+  /**
+   * Add title tag in svgs.
+   */
+  improveAccessibility() {
```

Comment:
this works fine, but I'm wondering why we don't put the title directly in the markup, instead of using javascript?

Reply:
mmh..i did it this way thinking that we couldn't do it otherwise, but now that i see the icon template it seems we can pass as_image and extra_accessibility['title'] to get it..so yes, i will do it like that